twitchio/ext/eventsub/models.py

```diff
@@ -145,12 +145,10 @@ class BaseEvent:
     __slots__ = ("_client", "_raw_data", "subscription", "headers")
 
     @overload
-    def __init__(self, client: EventSubClient, _data: str, request: web.Request):
-        ...
+    def __init__(self, client: EventSubClient, _data: str, request: web.Request): ...
```

Comment:
Black was run without specifying the `-l 120` line length.

Reply:
I have updated to remove nesting.

Reply:
Not sure why it updated them files on my machine:

I ran `black twitchio --line-lemngth 120 --verbose`

Purposefully copied the command out of the lint CI file to ensure i was using the same settings.